modules/dkan/dkan_dataset/modules/dkan_dataset_collections/dkan_dataset_collections.features.field_instance.inc

```diff
@@ -0,0 +1,68 @@
+<?php
```

Comment:
I wonder if it's a good idea to name something "field_collection" given the specific meaning that has in Drupal? Let's discuss naming.

Reply:
:+1:  Renamed to `field_dataset_collections`

Reply:
@ethanteague I think this isn't quite consistent with other dataset fields. I'd propose `field_dkan_ispartof_ref`

Reply:
:+1: